CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Unreleased
 
+- [BUGFIX] feat!: Add network instrumentation for async/await URLSession APIs. See [#1394][]
```

Comment:
I'd argue it's `IMPROVEMENT` not a `BUGFIX`.

Also, we can probably drop `feat!:` it wasn't used in the convention so far.

Reply:
fixed